Lib/Magnet/Extensions/NSEvent+keyCombo.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  NSEvent+keyCombo.swift
+//
+//  Magnet
+//  GitHub: https://github.com/clipy
+//  HP: https://clipy-app.com
+//
+//  Copyright © 2015-2020 Clipy Project.
+//
+
+import Cocoa
+import Sauce
+
+extension NSEvent.EventType {
+    fileprivate var isKeyboardEvent: Bool {
+        return [.keyUp, .keyDown].contains(self)
```

Comment:
I think it would be better to add `.flagsChanged` as well, but what do you think?

Reply:
At first I thought that'd fire also when you shift-click, but it would fire when the modifier key itself is hit. Makes sense, added it!